inc/css/blocks/class-posts-css.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Css handling logic for group.
+ *
+ * @package ThemeIsle\GutenbergBlocks\CSS\Blocks
+ */
+
+namespace ThemeIsle\GutenbergBlocks\CSS\Blocks;
+
+use ThemeIsle\GutenbergBlocks\Base_CSS;
+
+use ThemeIsle\GutenbergBlocks\CSS\CSS_Utility;
+
+/**
+ * Class Button_CSS
+ */
+class Posts_CSS extends Base_CSS {
+
+	/**
+	 * The namespace under which the blocks are registered.
+	 *
+	 * @var string
+	 */
+	public $block_prefix = 'posts-grid';
+
+	/**
+	 * Generate Button CSS
+	 *
+	 * @param mixed $block Block data.
+	 * @return string
+	 * @since   1.3.0
+	 * @access  public
+	 */
+	public function render_css( $block ) {
+		$css = new CSS_Utility( $block );
+
+		// TODO: find why this is not working
+		$css->add_item(
+			array(
+				'selector'   => ' .wp-block-themeisle-blocks-posts-grid-post-body',
+				'properties' => array(
+					array(
+						'property' => 'text-align',
+						'value'    => 'textAlign',
+					),
+					array(
+						'property' => 'justify-content',
+						'value'    => 'verticalAlign',
+					),
+				),
+			)
+		);
+
+		$css->add_item(
+			array(
+				'selector'   => ' .wp-block-themeisle-blocks-posts-grid .wp-block-themeisle-blocks-posts-grid-post-image img',
+				'properties' => array(
+					array(
+						'property' => 'border-radius',
+						'value'    => 'borderRadius',
+					),
+				),
+			)
+		);
+
+		$style = $css->generate();
+
+		return $style;
+	}
+}
```

Comment:
@HardeepAsrani , any idea what I am doing wrong here? This does not work.

Reply:
Discussed over Slack, but for future references, it happens because the Post block doesn't have the ID attribute.